dataprofiler/labelers/base_data_labeler.py

```diff
@@ -738,14 +734,11 @@ def fit(self, x, y, validation_split=0.2, labels=None, reset_weights=False,
         for i in range(epochs):
             results.append(self._model.fit(train_data, cv_data))
             if i < epochs - 1:
-                # shuffle input data, these are pandas.DataFrame or
-                # pandas.Series
-                shuffle_inds = np.random.permutation(cv_split_index)
-                train_data_x = x.loc[shuffle_inds].reset_index(drop=True)
-                train_data_y = y.loc[shuffle_inds].reset_index(drop=True)
-
-                # free memory
-                del shuffle_inds
+                # shuffle input data
+                train_data_x = copy.deepcopy(x)
+                train_data_y = copy.deepcopy(y)
+                np.random.shuffle(train_data_x)
+                np.random.shuffle(train_data_y)
```

Comment:
this change will now train on the whole dataset as oppose to the subset of training only (removing cv)
I think we should maintain the shuffle_inds for now

Reply:
Also shuffle_inds maintains x,y permutation matching

Reply:
resolved